packages/graphile-build-pg/src/plugins/makeProcField.js

```diff
@@ -379,11 +426,10 @@ export default function makeProcField(
         return memo;
       }, {});
       if (isMutation) {
-        const resultFieldName = getResultFieldName(
+        const resultFieldName = inflection.functionMutationResultFieldName(
           proc,
-          type,
-          rawReturnType,
-          proc.returnsSet
+          getNamedType(type).name,
```

Comment:
Pass the type, rather than its name; it gives greater flexibility.
```suggestion
          getNamedType(type),
```

Reply:
Done. c829caf